homeassistant/components/simplefin/sensor.py

```diff
@@ -40,6 +43,13 @@ class SimpleFinSensorEntityDescription(SensorEntityDescription):
         unit_fn=lambda account: account.currency,
         icon_fn=lambda account: account.inferred_account_type,
     ),
+    SimpleFinSensorEntityDescription(
+        key="age",
+        translation_key="age",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        value_fn=lambda account: dt_util.as_utc(account.balance_date),
```

Comment:
Should already be utc now
```suggestion
        value_fn=lambda account: account.balance_date,
```